src/components/app.tsx

```diff
@@ -235,6 +236,15 @@ export class App extends React.PureComponent<IProps, IState> {
                                  ? activityIndex + 1
                                  : undefined;
     sequenceActivity !== undefined && setQueryValue("sequenceActivity", sequenceActivity);
+    const backgroundImage = sequence?.background_image
+                              ? sequence.background_image
+                              : activity.background_image
+                                ? activity.background_image
+                                : undefined;
+    if (backgroundImage !== undefined) {
```

Comment:
Safer as `if (backgroundImage) {` since then we'll also skip cases where it is null, empty string, etc.